README.md

```diff
@@ -65,8 +65,8 @@ or to a Databricks Runtime interactive cluster (e.g. /sql/protocolv1/o/123456789
 
 ## Contributing
 
-See [CONTRIBUTING.md](CONTRIBUTING.md)
+See [CONTRIBUTING.md](../CONTRIBUTING.md)
```

Comment:
Both `README.md` and `CONTRIBUTING.md` are top level, this change is not required.

Reply:
fixed